requirements/dev.txt

```diff
@@ -179,7 +179,7 @@ packaging==20.9
     #   tox
 path.py==12.5.0
     # via edx-i18n-tools
-path==15.1.2
+path==16.0.0
```

Comment:
https://github.com/jaraco/path/blob/main/CHANGES.rst

> #200: `TempDir` context now cleans up unconditionally, even if an exception occurs.

is the only change in 15.1.2 to 16.0